---
content_type: page
description: This section provides summaries of each lecture session of the course.
learning_resource_types:
- Lecture Notes
ocw_type: CourseSection
title: Lecture Summaries
uid: 84ac4acd-1e05-7864-f01d-d885ea783ef1
---

{{< tableopen >}}
{{< theadopen >}}
{{< tropen >}}
{{< thopen >}}
WEEK #
{{< thclose >}}
{{< thopen >}}
TOPICS
{{< thclose >}}
{{< thopen >}}
LECTURE SUMMARIES
{{< thclose >}}

{{< trclose >}}

{{< theadclose >}}
{{< tropen >}}
{{< tdopen >}}
1
{{< tdclose >}}
{{< tdopen >}}
Introduction
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "Introduction" >}}{{< /anchor >}}During this session we will provide an overview of the semester. The instructor and students will introduce themselves, and we will discuss logistics to determine an optimal meeting time based on the availability of students who are interested in taking the course. We will discuss background issues in genomics and medicine that will be the basis for the course. We will also describe how to perform searches of the primary literature by using databases, including PubMed. We will also talk about how to analyze and critique scientific papers.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
2
{{< tdclose >}}
{{< tdopen >}}
Classical approaches to sequencing
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "Classical_approaches_to_sequencing" >}}{{< /anchor >}}Although the double-helical structure of DNA was first described in 1953, it took several decades to develop methods for the analysis of nucleic acid sequences. In 1977, Allan Maxam and Walter Gilbert developed a sequencing method that used chemical cleavage to create breaks in DNA followed by electrophoresis to separate the fragments and X-ray film analysis of the radioactively labeled DNA. Frederick Sanger developed an alternative methodology using primers and DNA polymerase in the presence of chain-terminating dideoxynucleotides. Sanger's method has been in use for over 25 years and was used in the original human genome initiative. In this class we will discuss these classical sequencing strategies, emphasizing elements of experimental design that have been incorporated into the "next generation" methods used for genome sequencing today.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
3
{{< tdclose >}}
{{< tdopen >}}
Contemporary sequencing methods
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "Contemporary_sequencing_methods" >}}{{< /anchor >}}There has been great demand for faster, lower cost technologies for DNA analysis, and a number of new methods termed "next generation" sequencing have been developed. In this session we will discuss new methodologies that have fueled improvements in sequencing over the past decade. We will also consider platform comparison studies to evaluate different sequencing methods that yield slightly different results, which may have important implications for the interpretation of human data.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
4
{{< tdclose >}}
{{< tdopen >}}
Single nucleotide polymorphisms
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "Single_nucleotide_polymorphisms" >}}{{< /anchor >}}Single-nucleotide polymorphisms (SNPs) are genomic alterations affecting individual DNA bases that can vary in frequency among different human populations. In some cases, SNPs can affect personal traits, susceptibility to disease or response to medicines, but more commonly they have no overt effect on phenotype. In this session we will talk about how SNPs are distributed throughout genes and intergenic regions and how these changes have been mapped across the human genome. The first paper that we will discuss shows how single-nucleotide changes were used to map a disease locus linked with two related neurodegenerative diseases. The next paper describes the nature of the mutation, a repeat expansion in a noncoding part of a gene with unknown function. These studies highlight the difficulty of translating findings from SNP association studies into molecular events that cause phenotypes in human populations.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
5
{{< tdclose >}}
{{< tdopen >}}
Copy number variation
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "Copy_number_variation" >}}{{< /anchor >}}In addition to changes at the nucleotide level, chromosomes can also have duplications or deletions of multigenic regions. Such structural changes are known as copy number variations (CNVs). We will discuss how copy number variation may be important in different human diseases. The first paper reports that a gene involved in a congenital motor neuron disease is duplicated in late onset motor neuron disease in adults. The second paper focuses on how whole genome sequencing approaches have been used to study patients with retinal disease and identifies both structural changes and new gene mutations.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
6
{{< tdclose >}}
{{< tdopen >}}
Tracing human populations
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "Tracing_human_populations" >}}{{< /anchor >}}While genomic variation is traditionally thought of as important for understanding traits and diseases, this information is also useful for anthropologists studying the migrations of human populations. The first paper that we will discuss describes the development of a GenoChip designed to clarify the relationships between extinct hominid populations and modern humans and to provide insight into human migratory history. The second paper traces human prehistory by sequence analysis of a bacterial parasite called _Helicobacter pylori_ that accompanied humans during migrations out of Africa. The genetic sequences of these bacteria differ from continent to continent as the result of genetic drift following migration of populations.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
7
{{< tdclose >}}
{{< tdopen >}}
Genome wide association studies
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "Genome_wide_association_studies" >}}{{< /anchor >}}Variations in numerous genes that are associated with particular traits or diseases have been identified using genome-wide association studies (GWAS). Human diseases may be multifactorial with small contributions from a large number of genes affecting to a person's risk of developing a given disorder. This week we will explore two papers that have used genetic association strategies to examine different blood disorders. In the first paper, a GWAS was performed to identify genetic modifiers in patients with thalassemia, a condition affects hemoglobin levels and causes anemia in patients. In the second paper, a genetic alteration in patients with Willebrand Disease, a disorder of blood clotting, was examined and experiments were performed to understand why a gene identified by GWAS may be involved in the pathophysiology of this disease.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
8
{{< tdclose >}}
{{< tdopen >}}
Field trip
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "Field_trip" >}}{{< /anchor >}}We will learn about the gene panels run using next generation sequencing at the {{% resource_link "ff885c38-ff40-4527-bae3-154b6b5b92da" "Harvard Medical School Center for Personalized Genetic Medicine" %}} and discuss the challenges faced by clinicians who are interpreting these studies. We will take a field trip to see next generation sequencing machines and how this technology is being applied in research and clinical diagnostics.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
9
{{< tdclose >}}
{{< tdopen >}}
Genomic analysis and targeted therapies
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "Genomic_analysis_and_targeted_therapies" >}}{{< /anchor >}}Chronic myelogenous leukemia (CML) is a hematologic malignancy characterized by a translocation between chromosomes 9 and 22 creating a fusion protein between the BCR and ABL genes. This gene fusion causes a abnormally active kinase to be produced, which promotes tumorigenesis by accelerating cell cycle kinetics. Drugs that inhibit this abnormal kinase activity such as imatinib can vastly improve survival rates for patients with this disease. The first paper that we will discuss uses Fluorescent In Situ Hybridization to define the chromosomal abnormality found in most patients with CML. Unfortunately, many of these tumors have additional mutations that allow them to grow in the presence of imatinib. A further understanding of these secondary mutations is described in the second paper.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
10
{{< tdclose >}}
{{< tdopen >}}
From rare mutation to mainstream drug target
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "From_raremutation_to_mainstream_drug_target" >}}{{< /anchor >}}Over 70 million Americans have high cholesterol, and many of these people take medicines such as statins to reduce the risk of coronary heart disease. In 2005, researchers at the University of Texas Southwestern Medical Center identified a healthy 40 year-old patient with unusually low levels of cholesterol in her bloodstream. By performing genetic and pedigree analysis she was found to be homozygous for a mutation in a gene later shown to regulate LDL levels in the blood. This gene encodes a protein called proprotein convertase subtilisin / kexin type 9 (PCSK9), which binds the LDL receptor preventing it from taking up cholesterol. The first paper describes the association between PCSK9 mutations and levels of low-density lipoprotein cholesterol in patients. Several pharmaceutical companies created antibody-based therapies to decrease PCSK9 so that LDL is removed from the bloodstream. The second paper describes how these monoclonal therapies work in patients who do not have the PCSK9 genetic mutation.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
11
{{< tdclose >}}
{{< tdopen >}}
Mosaics and chimeras: when genetic analysis goes astray
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "Mosaics_and_chimeras_when_genetic_analysis_goes_astray" >}}{{< /anchor >}}It has been traditionally assumed that healthy cells from the same person have identical genomes (with the exception of immune system and germ cell rearrangements). However, increasing evidence indicates that some genomic variation exists in differentiated tissues from the same person. The first study we will discuss investigates the genomes of six unrelated individuals and compares genomic content from different tissues, identifying a significant number of genomic changes between somatic tissues. The second paper addresses how mosaicism can be a problem in the diagnosis of genetic disorders by next generation sequencing.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
12
{{< tdclose >}}
{{< tdopen >}}
Crowdsourcing genetic studies
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "Crowdsourcing_genetic_studies" >}}{{< /anchor >}}In traditional studies, scientists analyze genome-wide association data in patients using bioinformatics programs to help interpret large datasets. A new approach allows people to submit their own genetic data, report their family histories and describe their traits via the internet. As indicated in the first paper, companies using this approach have validated previous genetic associations and identified new ones. Similarly, "crowdsourcing" or obtaining new information from a wide group of people has also been applied to the interpretation of genetic data. The second paper describes Open-Phylo, a game for performing multiple sequence alignments that has been played by over 10,000 people across the world. Are self-reported genetic association studies and human-computing tools the future for genomic research?
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
13
{{< tdclose >}}
{{< tdopen >}}
Understanding the causes of rare diseases
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "Understanding_the_causes_of_rare_diseases" >}}{{< /anchor >}}While the initial human genome sequencing projects focused on healthy individuals, there has been increased interest in performing whole-genome sequencing on families affected by rare diseases to understand the molecular causes of these conditions. While many studies do not reveal a single genetic mutation, this approach has been successful in helping both diagnosis and treatment of some conditions. In the first paper, whole genome sequencing was used to identify the causative mutations in several rare autosomal recessive conditions. A second study used whole-genome sequencing to identify a variant of dopa-responsive dystonia in a pair of twins, and this molecular diagnostic information guided their medical treatment.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
14
{{< tdclose >}}
{{< tdopen >}}
Final presentations
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "Final_presentations" >}}{{< /anchor >}}Students will give oral presentations as described above and we will also discuss the course in general.
{{< tdclose >}}

{{< trclose >}}

{{< tableclose >}}